src/tools/compiletest/src/runtest.rs

```diff
@@ -2042,10 +2050,24 @@ impl<'test> TestCx<'test> {
             if !is_rustdoc {
                 if let Some(ref linker) = self.config.linker {
                     rustc.arg(format!("-Clinker={}", linker));
+                } else {
+                    lld = self
+                        .config
+                        .target_rustcflags
+                        .as_ref()
+                        .map(|f| f.contains("-fuse-ld=lld"))
+                        .unwrap_or(false);
                 }
             }
         }
 
+        // Use a single thread for efficiency and a deterministic error message order
+        rustc.arg("-Zthreads=1");
```

Comment:
... and still is limited here.

Together with lld not obeying jobservers I don't see how there would be any benefit to using them in this narrow case. I can see how using them could make bootstrap better at a higher level though.